src/constants/markets.ts

```diff
@@ -422,6 +422,6 @@ export const OrderExecutionDeposit = Big6Math.fromFloatString('20')
 /**
  * @description When passed to trigger orders, this value will fully close the position.
  */
-export const TriggerOrderFullCloseMagicValue = 0n
+export const TriggerOrderFullCloseMagicValue = -(2n ** 63n)
```

Comment:
Do we need to handle the case where multiinvoker and manager values are different?